perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -621,7 +618,7 @@ working days (excluding weekends and bank holidays) after that date.
 
 =cut
 
-sub report_allowed {
+sub report_allowed_time {
```

Comment:
I'm not sure this name is an improvement tbh. Not that I can think of a better one :|

Reply:
within_two_working_days I guess describes what it does, the fact that gets assigned to "report_allowed" is immaterial